### ISSUE TITLE
Not calculating "Total %" correctly.

### ISSUE THREAD

**Issue by aon4o** [pestphp/pest-plugin-type-coverage#21]
## The problem:

I am currently working on a very big Laravel project and I believe type coverage is essential for avoiding bugs in the future.
"Very big" means that it has a lot of files - currently about 700 files are tested for type coverage and that number is going to get bigger in the future.

The problem I'm facing is that the type coverage plugin calculates the total % coverage wrong in my opinion.
For instance, all 700 files have 100% coverage, except one that has 67%.
In that case, the total percentages are "100.0 %", which means that if I don't scroll through the list of tested files I can't see if there is an error or not.

```
...
modules/Settings/Database/Seeders/SupportSettingsSeeder.php ..................... 100%  
modules/Settings/Database/factories/SupportSettingsFactory.php ..................... 100%  
modules/Settings/Database/factories/EmailTemplateFactory.php ....................... 100%
...  
modules/Settings/Entities/SupportSettings.php ...................................... pa34, pa35 67% 
...
modules/Orders/Entities/OrderItem.php .................................................. 100%  
modules/Orders/Entities/OrderStatus.php ............................................... 100%  
──────────────────────────────────────────────────
Total: 100.0 %
```

This also means that if I try to use the `--min` argument in a GitHub/Lab pipeline I cannot get an error and the job won't fail.

## Used versions:
- PHP: 8.2
- Laravel: 10.38
- Pest: 2.28
- Pest-Plugin-Laravel: 2.2
- Pest-Plugin-Type-Coverage: 2.5

Is there something I can do to fix that?
Thanks in advance!

**Comment by georgi-vdimitrov** [comment]
+1

**Comment by aon4o** [comment]
The issue was resolved by #18 .
